### PR TITLE
feat(tests): EOF folder structure re-org

### PR DESCRIPTION
**Pull request by marioevz**
## 🗒️ Description
Re-orgs all EOF tests under `./tests/prague/eip7692_eof_v1`, which is the meta-EIP that contains all the feature EIPs.

## 🔗 Related Issues
None

## ✅ Checklist
- [x] All: Set appropriate labels for the changes.
- [x] All: Considered squashing commits to improve commit history.
- [ ] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
- [x] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
- [x] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
- [x] Tests: Included the type and version of evm t8n tool used to locally execute test cases:  e.g., ref with commit hash or geth 1.13.1-stable-3f40e65.
- [x] Tests: Ran `mkdocs serve` locally and verified the auto-generated docs for new tests in the [Test Case Reference](https://ethereum.github.io/execution-spec-tests/main/tests/) are correctly formatted.

### CODE REVIEW

File: tests/prague/eip7692_eof_v1/tracker.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# EOF Testing Coverage Tracker
+
+## EIP-3540: EOF - EVM Object Format v1
+
+- [ ] Example Test Case 1
```
Comment (chfast) [inline tests/prague/eip7692_eof_v1/tracker.md:5]: Do we want multiple statuses?

- todo
- in ethereum/tests
- done

Comment (chfast) [reply tests/prague/eip7692_eof_v1/tracker.md:3]: Should we also write where the "done" test is?

Comment (marioevz) [reply tests/prague/eip7692_eof_v1/tracker.md:3]: Added!

**Review by winsvega — APPROVED** [review]
thats affect one of my ongoing prs. will have to adjust

**Pull request merged**